src/styles/blocks/stars.css

```diff
@@ -0,0 +1,28 @@
+.stars {
+  display: flex;
+  margin: 0;
+  padding: 0;
```

Comment:
I could be wrong, but it seems to me that this code is redundant

Reply:
I need this, otherwise I have wrong stars position